cs251tk/student/markdownify/markdownify.py

```diff
@@ -31,15 +34,20 @@ def markdownify(spec_id, username, spec, basedir, debug):
             result = process_file(filename, steps, options, spec, cwd, supporting)
             results['files'][filename] = result
 
+        # now we remove any compiled binaries
         try:
             for file in spec['files']:
                 os.remove('{}.exec'.format(file['filename']))
+        except FileNotFoundError:
+            pass
+
+        # and we remove any supporting files
+        try:
```

Comment:
This, I think, is actually an improvement; before, we would not remove any input files if we failed to remove any of the compiled files; now that that's split up, we'll handle the removes separately.